src/main/java/com/board/config/boardconfiggui/ui/utils/IpPinConfigListCell.java

```diff
@@ -0,0 +1,16 @@
+package com.board.config.boardconfiggui.ui.utils;
+
+import com.board.config.boardconfiggui.data.IpPinConfig;
+
+public class IpPinConfigListCell extends javafx.scene.control.ListCell<IpPinConfig> {
+  @Override
+  protected void updateItem(IpPinConfig item, boolean empty) {
+    super.updateItem(item, empty);
+    if (item == null || empty) {
+      setText("select");
```

Comment:
select is already available in the constants

Reply:
Updated to Constants.